packages/scatterplot/src/ScatterPlot.js

```diff
@@ -36,6 +36,9 @@ const ScatterPlot = ({
     enableGridX,
     enableGridY,
 
+    // markers
+    markers,
```

Comment:
The corresponding propType should be added (see https://github.com/plouc/nivo/blob/master/packages/line/src/props.js#L64).